src/transformers/member.ts

```diff
@@ -87,6 +89,6 @@ export interface DiscordenoMember {
   avatar?: bigint;
   /** The permissions this member has in the guild. Only present on interaction events. */
   permissions?: bigint;
-  /** when the user's timeout will expire and the user will be able to communicate in the guild again (up to 28 days in the future), set to null to remove timeout. Requires the `MODERATE_MEMBERS` permission */
-  communicationDisabledUntil?: number;
+  /** when the user's timeout will expire and the user will be able to communicate in the guild again, null or a time in the past if the user is not timed out */
+  communicationDisabledUntil?: number | null;
```

Comment:
```suggestion
  communicationDisabledUntil?: number;
```

fix the description also